website/api/secure-sdlc.md

```diff
@@ -0,0 +1,67 @@
+# RISC Zero Secure Development Life-cycle
+
+RISC Zero believes in a comprehensive risk-based approach to security throughout our product and software development life-cycle.
+
+This page provides an outline of the key practices that we utilize to harden our products and services against threats. These security techniques are continuously evolving as we learn from our experiences and adapt to new threats, and we encourage you to provide feedback or suggestions for improvement.
+
+The key mechanisms that drive our process evolution are:
+* External critique from customers, partners, and the security community
+* Escape analysis of production defects and postmortem analysis of security incidents
+* Identifying new potential activities based on industry frameworks including [NIST SSDF](https://csrc.nist.gov/Projects/ssdf), [ISO 27002:2022](https://www.iso.org/standard/75652.html), [BSIMM 14](https://www.synopsys.com/software-integrity/software-security-services/bsimm-maturity-model.html), [the ReKT test](https://www.risczero.com/blog/risc-zero-take-the-rekt-test), and [OWASP SAAAM](https://owasp.org/www-project-samm/).
+* Use of a maturity framework (known as FACTORS) to self-assess process maturity in the following areas:
```

Comment:
I'm not finding a reference for `FACTORS` after a quick google search. Can we add a link? 
```suggestion
* Use of a maturity framework (known as [FACTORS]) to self-assess process maturity in the following areas:
```